en/editor/publish/publish-huawei-ohos.md

```diff
@@ -0,0 +1,117 @@
+
+4. After the configuration is complete, connect to the Huawei device of the Harmony OS system, and then click **Build -> Build APP(s)/Hap(s) -> Build APP(s)** in the menu bar to execute the compilation process . Another way is to directly click the run button above **DevEco Studio** to execute the compile and run process.
+
+    ![run project](./publish-huawei-ohos/run-project.png)
+
+5. After compiling, notice the generated `.hap` file in the `native\engine\ohos\build\outputs\hap` directory of the project directory.
+
+    ![output](./publish-huawei-ohos/output.png)
+
+    The generation path of the `.hap` file for debugging is in the `native\engine\ohos\entry\build\outputs\hap` directory of the project directory.
+
+    ![output](./publish-huawei-ohos/debug-output.png)
+
+6. Please refer to the official [Release HarmonyOS Application Guide](https://developer.huawei.com/consumer/cn/doc/distribution/app/agc-harmonyapp-releaseharmonyapp#h1-1598338018957) documentation for the specific process of publishing to the Huawei App Market.
+
+### Signature file
+
+HarmonyOS applications use digital certificates (`.cer` files) and HarmonyAppProvision files (`.p7b` files) to ensure the integrity of the application. First, use **DevEco Studio** to generate a key and certificate request file, and then use the certificate request file to apply for a digital certificate and profile file for publishing/debugging in AppGallery Connect.
+
+#### Generate key and certificate request file
+
+In **DevEco Studio**, click **Build -> Generate Key and CSR** in the upper menu bar to generate a key (`.p12` file), and then generate a certificate request file (`.csr`). For details, please refer to the [Generate Key and Certificate Request File](https://developer.harmonyos.com/cn/docs/documentation/doc-guides/publish_app-0000001053223745#EN-CN_TOPIC_0000001154985553__section7209054153620) documentation.
+
+> **Note**: When specifying the file path, the file name needs to be added with the corresponding suffix, otherwise the submission will not be successful.
+
+#### Generate and download a digital certificate (`.cer` file)
+
+- Visit the [AppGallery Connect website](https://developer.huawei.com/consumer/cn/service/josp/agc/index.html), select **Users and Access -> Certificate Management**.
+
+  > **Note**: Currently, only invited developers can access the **Certificate Management** menu. To apply, please send the developer's name, application background, supported device type, whether there is a request for app listing, and Developer ID to `agconnect@huawei.com`. Huawei operations staff will arrange a docking staff within 1-3 working days . <br />The developer ID query method is: log in to the AGC website, click **My Project**, select any project, and find the `Developer ID` in **Project Settings -> General**.
```

Comment:
This can be deleted, now no longer need to apply, please help to delete the Chinese document also, thank you.